source/localizable/tutorial/ember-cli.md

```diff
@@ -131,5 +131,7 @@ or, for short:
 ember s
 ```
 
-If we navigate to [`http://localhost:4200`](http://localhost:4200), we'll see our brand new app displaying
-the text "Welcome to Ember".
+If we navigate to [`http://localhost:4200`](http://localhost:4200), we'll see the default welcome screen.
+Once we add our own `app/templates/application.hbs` file, the welcome screen will be replaced with our own content.
```

Comment:
@acorncom @locks should we add an additional mention that creating a route makes it go away too?